app/models/street_cafe.rb

```diff
@@ -11,5 +11,9 @@ def cafes_by_category(*args)
         where("category LIKE ? OR category LIKE ?", args[0], args[1])  
       end
     end
+
+    def find_duplicate_records
```

Comment:
I am using this method to find duplicate records if the csv import task is used more than once with the same csv file. This allows the import cafe method to be more flexible and could allow someone to import multiple csv files without deleting records from the first one. I originally was using a StreetCafe.destroy_all